spec/features/venues/delete_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'deleting a city', type: :feature do
```

Comment:
Please update the feature spec name to replace the example text with one relevant to the spec